src/bci_build/package/globals.py

```diff
@@ -0,0 +1,181 @@
+"""This module contains global variables and enums used in the BCI Dockerfile generator."""
+from __future__ import annotations
+
+import datetime
+import enum
+from typing import List
+
+
+_BASH_SET = "set -euo pipefail"
```

Comment:
Since this is an exported symbol, it shouldn't be private
```suggestion
BASH_SET = "set -euo pipefail"
```

Reply:
done, that needs followup fixes.